Crashlytics/Crashlytics/Models/Record/FIRCLSRecordSignal.m

```diff
@@ -25,7 +25,7 @@ - (instancetype)initWithDict:(NSDictionary *)dict {
     _name = dict[@"name"];
     _code_name = dict[@"code_name"];
     _err_no = [dict[@"err_no"] unsignedIntegerValue];
-    _time = [dict[@"time"] unsignedIntegerValue];
+    self.time = [dict[@"time"] unsignedIntegerValue];
```

Comment:
This should be changed back to _time since we are in the init

Reply:
For some reason we can't because time is defined in the superclass